src/verso/Verso/Syntax.lean

```diff
@@ -6,6 +6,20 @@ Author: David Thrane Christiansen
 
 import Verso.SyntaxUtils
 
+/-!
+
+This file contains an internal syntax that's used to represent Verso documents. It's primarily
+intended for use in quasiquotations that match against the output of the Verso parser, though it
+does serve to document the expected format of the syntax data produced by the parser. A later step
+transforms the syntax in this module into the actual syntax of Verso documents, potentially invoking
```

Comment:
At first read this sounds backwards; are you saying that the syntax _parsers_ here are automatically transformed into parsers for verso documents? Or that there is a delaborator that converts the syntax here into the document syntax? Or did you mean that the document syntax is transformed into this internal syntax?

Reply:
The latter - I'll clarify.